config/initializers/booking_bug_config.rb

```diff
@@ -0,0 +1,24 @@
+module Config
+  cattr_accessor :booking_bug
+
+  class BookingBug
+    attr_accessor :environment, :company_id, :api_key, :app_id, :email, :password
+
+    def initialize
+      yield(self) if block_given?
+    end
+
+    def attributes
+      instance_variable_names.map { |n| n.gsub(/@/, '') }
```

Comment:
Why are we providing configuration values that we're then modifying? We should be be able to provide values that can be used directly.